Software/bsw/static/COM/CanIf/inc/CanIf_Types.h

```diff
@@ -147,27 +147,15 @@ Note: If CanIfTxPduTriggerTransmit is not specified or FALSE, no upper
 layer modules have to be configured for Trigger Transmit. Therefore,
 <User_TriggerTransmit>() will not be called and CanIfTxPduUserTxConfirmationUL
 as well as CanIfTxPduUserTriggerTransmitName need not to be configured. */
-#if(CANIF_TX_PDU_TRIGGER_TRANSMIT==STD_ON)
-	typedef uint8 CanIfTxPduUserTxConfirmationULType;
-	#define CAN_NM							((CanIfTxPduUserTxConfirmationULType)0x00U)
-	#define CAN_TP							((CanIfTxPduUserTxConfirmationULType)0x01U)
-	#define CAN_TSYN						((CanIfTxPduUserTxConfirmationULType)0x02U)
-	#define CDD								((CanIfTxPduUserTxConfirmationULType)0x03U)
-	#define J1939NM							((CanIfTxPduUserTxConfirmationULType)0x04U)
-	#define J1939TP							((CanIfTxPduUserTxConfirmationULType)0x05U)
-	#define PDUR							((CanIfTxPduUserTxConfirmationULType)0x06U)
-	#define XCP								((CanIfTxPduUserTxConfirmationULType)0x07U)
-#endif
 typedef uint8 CanIfTxPduUserTxConfirmationULType;
-#define CAN_NM_TX_CONFIRMATION							((CanIfTxPduUserTxConfirmationULType)0x00U)
-#define CAN_TP_TX_CONFIRMATION							((CanIfTxPduUserTxConfirmationULType)0x01U)
-#define CAN_TSYN_TX_CONFIRMATION						((CanIfTxPduUserTxConfirmationULType)0x02U)
-#define CDD_TX_CONFIRMATION								((CanIfTxPduUserTxConfirmationULType)0x03U)
-#define J1939NM_TX_CONFIRMATION							((CanIfTxPduUserTxConfirmationULType)0x04U)
-#define J1939TP_TX_CONFIRMATION							((CanIfTxPduUserTxConfirmationULType)0x05U)
-#define PDUR_TX_CONFIRMATION							((CanIfTxPduUserTxConfirmationULType)0x06U)
-#define XCP_TX_CONFIRMATION								((CanIfTxPduUserTxConfirmationULType)0x07U)
-
+#define CAN_NM							((CanIfTxPduUserTxConfirmationULType)0x00U)
+#define CAN_TP							((CanIfTxPduUserTxConfirmationULType)0x01U)
+#define CAN_TSYN						((CanIfTxPduUserTxConfirmationULType)0x02U)
+#define CDD								((CanIfTxPduUserTxConfirmationULType)0x03U)
+#define J1939NM							((CanIfTxPduUserTxConfirmationULType)0x04U)
+#define J1939TP							((CanIfTxPduUserTxConfirmationULType)0x05U)
+#define PDUR							((CanIfTxPduUserTxConfirmationULType)0x06U)
+#define XCP								((CanIfTxPduUserTxConfirmationULType)0x07U)
```

Comment:
We should be specific as there is a Tx Confirmation and Rx indication so it should be having an indication whether it's CAN_NM_TX_CONFIRMATION or CAN_NM_RX_INDICATION
that's the reason for having this parameter named like that so please use this name